src/Conda.jl

```diff
@@ -192,7 +192,11 @@ end
 const PkgOrPkgs = Union{AbstractString, AbstractVector{<: AbstractString}}
 
 "Install a new package or packages."
-function add(pkg::PkgOrPkgs, env::Environment=ROOTENV; channel::AbstractString="")
+function add(pkg::PkgOrPkgs; env::Environment=ROOTENV, channel::AbstractString="")
```

Comment:
This is a breaking change.  You should add:
```jl
@deprecate add(pkg::PkgOrPkgs, env::Environment) add(pkg, env=env)
```
and similar so that callers using the old API still function, albeit with a warning message.